src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -305,6 +323,11 @@ public Command OnErrorLine(Action<string> handler)
             return this;
         }
 
+        public bool IsUnsafe
```

Comment:
I introduced a comment for the backing field: https://github.com/cdmihai/cli/commit/dac549b3b7c385c4c71ddddeeaf64e9fd0c3d821#diff-8a6abf84230f41f4be1751a274a1df3fR32

Is that alright? If yes, I'll move it to the property.